server/chat-plugins/room-faqs.ts

```diff
@@ -120,6 +120,12 @@ export const commands: ChatCommands = {
 
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(Chat.formatText(roomFaqs[room.roomid][topic], true));
+		if (!this.broadcasting && user.can('ban', null, room, 'rfaq')) {
+			const arg = roomFaqs[room.roomid][topic];
+			this.sendReplyBox(
+				Utils.html`<code>/addfaq ${topic}, ${arg} </code>`
+			);
```

Comment:
```suggestion
			Chat.getReadmoreCodeBlock(Utils.html`/addfaq ${topic}, ${arg}`);
```
Actually, no. This should use a readmore block, I'm fairly sure.

Reply:
If i do that, he break link http://prntscr.com/xa782h